docs/OV_Runtime_UG/performance_hints.md

```diff
@@ -0,0 +1,146 @@
+# High-level Performance Hints {#openvino_docs_OV_UG_Performance_Hints}
+
+Each of the OpenVINO's [supported devices](supported_plugins/Supported_Devices.md) offers low-level performance settings. Tweaking this detailed configuration requires deep architecture understanding.
+Also, while the performance may be optimal for the specific combination of the device and the inferred model, the resulting configuration is not necessarily optimal for another device or model.
+The OpenVINO performance hints are the new way to configure the performance with the _portability_ in mind. 
+
+The hints also "reverse" the direction of the configuration in the right fashion: rather than map the application needs to the low-level performance settings, and keep an associated application logic to configure each possible device separately, the idea is to express a target scenario with a single config key and let the *device* to configure itself in response.
+As the hints are supported by every OpenVINO device, this is completely portable and future-proof solution. 
```

Comment:
```suggestion
The hints also "reverse" the direction of the configuration in the right fashion: rather than map the application needs to the low-level performance settings, and keep an associated application logic to configure each possible device separately, the idea is to express a target scenario with a single config key and let the *device* to configure itself in response.
As the hints are supported by every OpenVINO device, this is a completely portable and future-proof solution. 
```